jdaviz/configs/specviz/plugins/viewers.py

```diff
@@ -47,13 +47,48 @@ def __init__(self, *args, **kwargs):
         self._offscreen_lines_marks = OffscreenLinesMarks(self)
         self.figure.marks = self.figure.marks + self._offscreen_lines_marks.marks
 
+        self.label_mouseover = None
+        self.add_event_callback(self.on_mouse_or_key_event, events=['mousemove', 'mouseenter',
+                                                                    'mouseleave'])
         self.state.add_callback('show_uncertainty', self._show_uncertainty_changed)
 
         self.display_mask = False
 
         # Change collapse function to sum
         self.state.function = 'sum'
 
+    def on_mouse_or_key_event(self, data):
+
+        if self.label_mouseover is None:
+            if 'g-coords-info' in self.session.application._tools:
+                self.label_mouseover = self.session.application._tools['g-coords-info']
+            else:
+                return
+
+        if data['event'] == 'mousemove':
+            # Extract data coordinates - these are pixels in the reference image
+            x = data['domain']['x']
+            y = data['domain']['y']
+
+            if x is None or y is None:  # Out of bounds
+                self.label_mouseover.pixel = ""
+                self.label_mouseover.reset_coords_display()
+                self.label_mouseover.value = ""
+                return
+
+            fmt = 'x={:+10.5e} y={:+10.5e}'
+            self.label_mouseover.pixel = fmt.format(x, y)
```

Comment:
This PR implements what the user is asking for, but I'm wondering if we that's what we want. This proposed specviz behavior is a bit different from the [imviz mousover behavior](https://github.com/spacetelescope/jdaviz/blob/f14b99c8a834eabf716987e90870cbe77f0b4939/jdaviz/configs/imviz/plugins/viewers.py#L121-L132) which displays the intensity at the pixel under the cursor. Wouldn't it be more consistent with the other helpers to display the x value of the mouse, and y value of the data/spectrum at the x of the mouse? Otherwise the toolbar `x, y` coordinates are just telling you something that you can already read from the axis labels.

Reply:
That requires additional logic (how to determine which spectrum to read, whether its closest in x or closest in x/y, whether to interpolate, how to depict the point on the spectrum being read), so we decided to implement just mouseover coordinates for now and reconsider more advanced functionality later.  Although maybe we should leave the issue open since it is requesting more?

Reply:
We had a whole meeting discussing this. If you want to display value of the data, then it becomes complicated if you have multiple spectra and subsets all visible at the same time.

Reply:
OK, this PR now does not close the issue.